spec/controllers/idv/personal_key_controller_spec.rb

```diff
@@ -196,6 +219,16 @@ def index
 
         expect(response).to redirect_to idv_in_person_ready_to_verify_url
       end
+
+      it 'logs key submitted event' do
+        patch :update
+
+        expect(@analytics).to have_logged_event(
+          'IdV: personal key submitted', address_verification_method: nil,
+                                         deactivation_reason: nil,
+                                         proofing_components: nil
```

Comment:
ditto all same or all different
```suggestion
          'IdV: personal key submitted',
          address_verification_method: nil,
          deactivation_reason: nil,
          proofing_components: nil,
```